src/main/java/org/candlepin/resource/OwnerResource.java

```diff
@@ -1239,8 +1240,14 @@ public Stream<PoolDTO> listOwnerPools(
 
         // Process the filters passed for the attributes
         PoolFilterBuilder poolFilters = new PoolFilterBuilder();
-        for (KeyValueParamDTO filterParam : attrFilters) {
-            poolFilters.addAttributeFilter(filterParam.getKey(), filterParam.getValue());
+        for (String filterParam : attrFilters) {
+            String[] keyValue = filterParam.split(":");
+            if (keyValue.length == 2) {
+                poolFilters.addAttributeFilter(keyValue[0], keyValue[1]);
+            }
+            else {
+                throw new CandlepinParameterParseException("name:value");
```

Comment:
it does because you have mapper for it to map it to 400. i add another param so user now know which parsed attributed is failing. 
"""
Runtime Error failed to parsed: support_level;Supurb, expected format: key:value at org.candlepin.resource.OwnerResource.listOwnerPools:1,249
"""
and here you can see the translation
https://github.com/candlepin/candlepin/blob/003e21e8deb95c4a0f976d5a5e8faf698f9e8dfa/src/main/java/org/candlepin/exceptions/mappers/CandlepinExceptionMapper.java#L103-L107